File: C++/Easy/FYI/FYI.cpp
#include <iostream>
#include <fstream>
#include <string>
using namespace std;

int main(int argc, char const *argv[])
{
    string input;

    /*
    ifstream inFile;
    
    inFile.open("C:/Users/zeb_a/Documents/Kattis/C++/Easy/FYI/fyi/a002.in", ios::in);

    if(!inFile){
        cout << "Unable to open file";
        exit(1);
    }
    string i;
    while(getline(inFile, i)){
        input += i;
        //cout << input;
    }
*/

while (cin >> input)
{
   string firstThree = input.substr(0,3);


    int result = firstThree.compare("555");

    if(result == 0) cout << "1" << endl;
    else cout << "0" << endl;
}

    


    //inFile.close();
    
    return 0;
}

File: Python/Easy/FYI/FYI.py
import sys

#inputFile = "C:/Users/zeb_a/Documents/Kattis/Python/Easy/FYI/fyi/a002.in"
#input = open(inputFile, "r").read()




for i in sys.stdin:
    input = i
    firstThree = ""

    for x in range(0,3):
        firstThree += input[x]

    if(firstThree == "555"): print("1")
    else: print("0")

